aiogram/dispatcher/filters/builtin.py

```diff
@@ -691,3 +691,17 @@ def __init__(self, is_forwarded: bool):
 
     async def check(self, message: Message):
         return bool(getattr(message, "forward_date")) is self.is_forwarded
+
+
+class ChatTypeFilter(BoundFilter):
+    key = 'chat_types'
+
+    def __init__(self, chat_types: typing.List[ChatType]):
```

Comment:
why is `chat_types` type restricted to `typing.List[ChatType]`, why not `Set` or any other container?

I'd use `Container[ChatType]` and given [@uburuntu's commentary](https://github.com/aiogram/aiogram/pull/356/files#r442129735). To reflect the current API, I suppose it'd be nice if we could:

0. rename `chat_types` to a singular `chat_type`
1. handle single value passed as a string, in the end having small instance check like...
```python3
self.required_types: Set[str]
if isinstance(chat_type, str):
    self.required_types = {chat_type}
else:
    self.required_types = set(chat_type)
```

what do you think?

Reply:
Okay, yeap, I've implemented the single value passing. Take a look at https://github.com/aiogram/aiogram/pull/356/commits/768491d97604e9e25d53f432fe12c597121ee88e, please